src/app/pages/OnDemandTvPage/VideoPlayer/index.jsx

```diff
@@ -49,7 +54,7 @@ const VideoPlayer = ({ assetId, masterBrand, imageUrl, isExpired }) => {
     translations,
   );
 
-  if (isExpired) {
+  if (!isEpisodeAvailable) {
```

Comment:
this naming seems odd. it's like a question rather than a statement

Reply:
episodeAvailable?

Reply:
"is" in boolean variable names is a good idea so `episodeIsAvailable` would be better

Reply:
One day, one day I will get the order correct ;)